Nikita_Korzhavin/hamburger.js

```diff
@@ -0,0 +1,51 @@
+<<<<<<< HEAD
+var Product = require('./api/products.js');
+var CombinedProduct = require('./api/combinedProducts');
+
+function Hamburger(options) {
+  this.SIZE_SMALL = new Product(50, 20, 'small');
```

Comment:
Hmm, actually I don't like the idea to store such constants inside the class

Reply:
fixed